models/step_analytics.go

```diff
@@ -6,15 +6,19 @@ import (
 
 // StepAnalytics ...
 type StepAnalytics struct {
-	AppSlug     string        `json:"-"`
-	BuildSlug   string        `json:"-" track:"build_slug"`
-	StepID      string        `json:"step_id" track:"step_id"`
-	StepTitle   string        `json:"step_title" track:"step_title"`
-	StepVersion string        `json:"step_verion" track:"step_verion"`
-	StepSource  string        `json:"step_source" track:"step_source"`
-	Status      string        `json:"status" track:"status"`
-	StartTime   time.Time     `json:"start_time" track:"start_time"`
-	Runtime     time.Duration `json:"run_time" track:"run_time"`
+	// connect step analytics to Bitrise builds
+	AppSlug   string `json:"-"`
+	BuildSlug string `json:"-" track:"build_slug"`
+
+	StepID      string            `json:"step_id" track:"step_id"`
+	StepTitle   string            `json:"step_title" track:"step_title"`
+	StepVersion string            `json:"step_verion" track:"step_verion"`
```

Comment:
Does `step_verion` have a typo by design, or something that slipped through?

Reply:
good catch, fix in the next commit